src/data/externalTutorials.json

```
@@ -30,6 +30,37 @@
     "lang": "en",
     "publishDate": "09/09/2021"
   },
+  {
+    "url": "https://www.youtube.com/watch?v=ZurrDzuurQs",
+    "title": "Learn Dapptools | Code like MakerDAO & the DAI team",
+    "description": "We look at how to make all you wonderful web3, blockchain, and smart contract developers even BETTER by showing you how to use dapptools! Article link: https://medium.com/@patrick.collins_58673/how-to-use-dapptools-code-like-makerdao-fed9909d055b",
```

Would consider removing the article link here. It isn't clickable on the tutorial card and also not easy to select since the whole card is clickable. Since this article is linked on the YouTube video itself, I would consider just removing it. 
```suggestion
    "description": "We look at how to make all you wonderful web3, blockchain, and smart contract developers even BETTER by showing you how to use dapptools!",
```